eox_hooks/tests/test_receivers.py

```diff
@@ -0,0 +1,55 @@
+"""This file contains all the test for the receivers.py file.
+
+Classes:
+    SingalsTestCase: Test Receivers.
+"""
+from django.conf import settings
+from django.test import TestCase, override_settings
+from mock import Mock, patch
+
+from eox_hooks.receivers import hooks_handler
+
+
+class TestReceivers(TestCase):
```

Comment:
it is missing the test case when the dispatch doesn't have a value which starts with `eox-hooks:`